src/pkg/control/options.go

```diff
@@ -112,3 +112,19 @@ type Toggles struct {
 	// graph API. This flag is only relevant for exchange backups for now
 	DisableConcurrencyLimiter bool `json:"disableConcurrencyLimiter,omitempty"`
 }
+
+// ---------------------------------------------------------------------------
+// Repo Maintenance flags
+// ---------------------------------------------------------------------------
+
+type Safety string
+
+const (
+	FullSafety Safety = "full"
+	NoSafety   Safety = "none"
```

Comment:
These are understandable comparison to their kopia usage, but don't have much context when read on their own.  Further, I'm not sure we want all services to observe a common notion of `safety`, as those concepts could be critically different between services.

It's a tough call, because I don't want to expose kopia configuration at higher levels, but kopia _does_ feel like the correct owner here.  I think it would be reasonable to have it export these two enums.  Understandably, if we don't want to do that, I think some other form of namespacing like `control.Maintenance{ Safety: enum }` or `control.Repository{ MaintenanceSafety: enum }` would help identify ownership, and prevent this from accidentally spilling over into unintended spaces.